### ISSUE TITLE
MLflow deployment example

### ISSUE THREAD

**Issue by wd60622** [pymc-labs/pymc-marketing#901]
No description provided.

**Comment by juanitorduz** [comment]
💯 !

**Comment by louismagowan** [comment]
@wd60622 I have some code written on this already which I can share if you wanna do the PR together 😁

**Comment by wd60622** [comment]
> @wd60622 I have some code written on this already which I can share if you wanna do the PR together 😁

Yeah, sounds great. Maybe drop what you have here or let's create a draft PR. 

I against mlflow being a required dependency but think some type of helper submodule would be great. I see
`pip install pymc-marketing[mlflow]` in our future

**Comment by louismagowan** [comment]
Ok cool, bear with me ⏳ :grin

**Comment by louismagowan** [comment]
Draft PR can be found [here](https://github.com/pymc-labs/pymc-marketing/pull/912)
